### PR TITLE
Egyptian Fraction Algorithm added

### PR DESCRIPTION
**Pull request by parjanyaacoder**
Added Egyptian Fraction Algorithm.
Fixes Issue#113

### CODE REVIEW

File: EgyptianFractions.cpp [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{  int numerator,denominator,quotient;
+    cout<<"Enter numerator and denominator";
```
Comment (Bhavnaharitsa) [inline EgyptianFractions.cpp:22]: no need elaborated variables, you can use short forms avoiding single letter

File: EgyptianFractions.cpp [hunk 2]
```diff
@@ -0,0 +1,37 @@
+#include <iostream>
```
Comment (Bhavnaharitsa) [inline EgyptianFractions.cpp:16]: add algorithm name, description, sample input-output

**Comment by Bhavnaharitsa** [comment]
Automatically add it in the Readme

**Comment by Bhavnaharitsa** [comment]
You have reopened this pull request. Are you sure you want to keep the changes?
Else, I will not review

**Comment by parjanyaacoder** [comment]
Actually closing this pull request was just a mistake.I want to keep the changes so please review this pull request.

File: EgyptianFractions.cpp [hunk 3]
```diff
@@ -35,3 +41,13 @@ int main()
 
     return 0;
 }
+/* Sample Input
```
Comment (Bhavnaharitsa) [inline EgyptianFractions.cpp]: push this to top

File: README.md [hunk 4]
```diff
@@ -1,30 +1,34 @@
-# nwoc_algorithms
-This is a library of various algorithms that are used in competitive programming.
```
Comment (Bhavnaharitsa) [inline README.md:2]: why remove?

File: EgyptianFractions.cpp (hunk 3 above)
Comment (parjanyaacoder) [reply EgyptianFractions.cpp]: Done

File: README.md (hunk 4 above)
Comment (parjanyaacoder) [reply README.md:2]: With everyone making pull requests and updating the readme my pull request was showing some conflicts.So I replaced some data in readme and added link to the new Egyptian Fraction Algorithm in the readme. I did this to resolve the conflicts.

File: README.md [hunk 5]
```diff
@@ -1,30 +1,34 @@
...
+### Sample algorithm
+[disjoint_set_union.cpp](Algorithms/disjoint_set_union.cpp) <br />
+[Floyd_cycle_detection.cpp](Algorithms/Linked_List/Floyd_cycle_detection.cpp)
+
+### Bellman Ford Algorithm
+ [Bellman Ford Algorithm](Bellman_Ford/BellmanFord.cpp)
+(Algorithms/.cpp)
+
+### Greedy Algorithm
+[Egyptian Fraction](https://github.com/NJACKWinterOfCode/nwoc_algorithms/pull/116/commits/a660bb5ca89951b744ae27ebc808c6e032ea6053)(Algorithms/EgyptianFractions.cpp)
```
Comment (Bhavnaharitsa) [inline README.md]: This does not give link
Look in rich text

Comment (parjanyaacoder) [reply README.md]: I have updated readme now with correct link so please review it. I think it will work fine now

File: README.md [hunk 6]
```diff
@@ -25,7 +25,7 @@ This is a library of various algorithms that are used in competitive programming
 [Kruskal](Algorithms/Graphs/Kruskal's%20Algorithm.cpp)
 
 ### Greedy Algorithm
-[Egyptian Fraction](https://github.com/NJACKWinterOfCode/nwoc_algorithms/pull/116/commits/a660bb5ca89951b744ae27ebc808c6e032ea6053)(Algorithms/EgyptianFractions.cpp)
+[Egyptian Fraction](Algorithms/EgyptianFractions.cpp)
```
Comment (Bhavnaharitsa) [inline README.md]: page not found

File: README.md [hunk 7]
```diff
@@ -25,7 +25,7 @@ This is a library of various algorithms that are used in competitive programming
 [Kruskal](Algorithms/Graphs/Kruskal's%20Algorithm.cpp)
 
 ### Greedy Algorithm
-[Egyptian Fraction](Algorithms/EgyptianFractions.cpp)
+[Egyptian Fraction](EgyptianFractions.cpp)
```
Comment (parjanyaacoder) [inline README.md:28]: Ma,am Please see this this is the new readme file

**Pull request merged**